src/backend.h

```diff
@@ -27,13 +27,14 @@
 #include <fstream>
 
 
-const uint16_t MAX_RETRIES                 = 100;
-const uint8_t  DEFAULT_V4L2_FRAME_BUFFERS  = 4;
-const uint16_t DELAY_FOR_RETRIES           = 50;
-const int      POLLING_DEVICES_INTERVAL_MS = 5000;
+const uint16_t MAX_RETRIES                         = 500;
+const uint8_t  DEFAULT_V4L2_FRAME_BUFFERS          = 4;
+const uint16_t DELAY_FOR_RETRIES                   = 10;
```

Comment:
Not sure about the other changes but please note that "DELAY_FOR_RETRIES" is used in other places in LRS SDK. (for example on l500-device.cpp).
Changing the value will effect other processes.

Reply:
Thank you for your comment. I overlooked this because I initially made the commit in this PR based on the master branch and then cherry-picked it on top of the development branch for submitting the PR. Based on grep-ing the source, it seems that "DELAY_FOR_RETRIES" was not used anywhere else in master, but in development, it has been introduced into three pieces of identically-looking code which start on these lines:
https://github.com/IntelRealSense/librealsense/blob/fc007cef1b9bb4f68dfdb540fcbea70aac46ebaa/src/l500/l500-device.cpp#L357
https://github.com/IntelRealSense/librealsense/blob/fc007cef1b9bb4f68dfdb540fcbea70aac46ebaa/src/ivcam/sr300.cpp#L350
https://github.com/IntelRealSense/librealsense/blob/fc007cef1b9bb4f68dfdb540fcbea70aac46ebaa/src/ds5/ds5-device.cpp#L147

Based on a quick look at these pieces of code, my impression is that the exact delay there probably does not matter much, since the code only seems to query a `bool` behind a `mutex` based on this delay and keeps the overall query duration constant. But I don't know which delay would be most appropriate here; I could easily revert it to 50 milliseconds if desired.

My reasoning for reducing the delay in the other place (the retry interval for USB commands) was that with a D415 device with slightly non-current firmware, and the command to read out the device clock, there seemed to be three distinct outcomes in the code that I tested:
1. The response to the command could be read immediately
2. A single retry (without any delay) was needed to get the response after getting EBUSY in the first try
3. It took about 82 +- 1 milliseconds to get the response, with EBUSY being returned in the meantime

Waiting 50 milliseconds as the original code does seemed to cause a lot of unnecessary waiting in cases 2 and 3. The changed retry strategy in this PR should be able to get the response as soon as possible in cases 1 and 2, and reduce the waiting in case 3.

This is just based on observations though; I do not have any previous experience with low-level USB handling.